src/Features/Admin/Categories/AddCategory/index.js

```diff
@@ -40,7 +40,7 @@ const AddCategory = () => {
     const { validateArray, isValid } = validateOnSubmit(fields, true);
     setFields(validateArray);
     if (isValid) {
-      const name = fields.map(({ value }, index) => value);
+      const name = fields.map(({ value }) => value);
```

Comment:
when we are giving name[0] below why do we need to do the map over fileds array ??